botorch/models/approximate_gp.py

```diff
@@ -7,24 +7,11 @@
 r"""
 References
 
-.. [burt2020svgp]
```

Comment:
Citations are global so these don't need to be (and can't be) defined in both `approximate_gp.py` and `inducing_point_allocators.py`